src/routes.ts

```diff
@@ -83,6 +83,38 @@ export default [
       ]
     }
   },
+  {
+    path: '/correction?u=client',
```

Comment:
These are internal routes for AR/COA/COD corrections. They should not need to change.